src/Datadog.Trace.ClrProfiler.Managed/Instrumentation.cs

```diff
@@ -59,6 +61,8 @@ public static void Initialize()
 
             try
             {
+                TryLoadVendorPlugin();
+
                 // ensure global instance is created if it's not already
```

Comment:
One thing that worries me is that if in this sentence. This should be the first bootstrapping point...

Reply:
But it is not. You have to put the `TryLoadVendorPlugin();` in `Tracer` constructor AFAIK...

Reply:
I tried to avoid exactly that, then plugins will be directly connected to `Tracer` instead of global scope.

Reply:
This is on the loading path, so it will be called on startup before anything else, but theoretically you could always call this later to ensure initialization

Reply:
@zacharycmontoya So this comment is misleading? Because how I understand your statement is that this is the first place where the `Tracer` instance would be created.

Reply:
@pellared well the trick is that, it's created on static call. I tracked this place as a first static call. However reading the comment, I can't be 100% sure that this is the same for every platform. I was just running Asp.Net Core on console. Another point is if it's protecting a rare race condition somewhere etc.

Reply:
Yes I suppose the comment is misleading. The first time the `Tracer` instance is created will be here

Reply:
@RassK Can you fix the comment please?